app/controllers/catkeys_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class CatkeysController < ApplicationController
-  before_action :load_and_authorize_resource
+  load_and_authorize_resource :cocina, parent: false, class: 'Repository', id_param: 'item_id'
```

Comment:
Can you explain this?

Reply:
It is a method in the Cancancan gem that we use for authorization: https://github.com/CanCanCommunity/cancancan/blob/develop/docs/controller_helpers.md#authorize_resource-load_resource-load_and_authorize_resource

Reply:
Thanks. Ruby magic.